app/views/questions/show.html.erb

```diff
@@ -0,0 +1,68 @@
+<h2><%= @question.question_title %></h2>
+<p><%= @question.content %></p>
+
+<% if logged_in? %>
+  <%= form_for @question, url: question_votes_path(@question), method: :post do |f| %>
```

Comment:
Why this form_for question if you're creating a vote? 
You could do form_for @question.votes.new